src/libraries/Microsoft.PowerFx.Connectors/Public/ConnectorRelationships.cs

```diff
@@ -0,0 +1,36 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+using System.Collections.Generic;
+using System.Linq;
+
+namespace Microsoft.PowerFx.Connectors
+{
+    public class ConnectorRelationships
```

Comment:
>public [](http://example.com/codeflow?start=4&length=6)

can we keep internal.  (will make it easier to change later)

Reply:
(I renamed it to TabularRelationships)

This one is intentional as expose relationships in this object which is supposed to be our neutral representation (not SF, ZD, SQL specific) for relationships

Reply:
Can we start with just exposing RecordType, and expose this later as we get clearer scenarios?